ee/clickhouse/queries/groups_join_query.py

```diff
@@ -58,3 +61,34 @@ def get_join_query(self) -> Tuple[str, Dict]:
             params[var] = group_type_index
 
         return "\n".join(join_queries), params
+
+    def get_filter_query(self, group_type_index: GroupTypeIndex) -> Tuple[str, Dict]:
+        var = f"group_index_{group_type_index}"
```

Comment:
👍

Reply:
We'll most probably re-use this function in cohort by groups